proposals/tsc-voting-procedures.md

```diff
@@ -0,0 +1,199 @@
+# TSC Voting Procedures Proposal
+
+## Overview
+
+This proposal establishes standardized voting procedures for the Hiero Technical Steering Committee (TSC) to improve decision-making efficiency while maintaining transparency and adherence to charter requirements.
+
+## Voting Methods
+
+### 1. In-Meeting Votes
+- **When Used**: For decisions during TSC meetings with quorum present
+- **Threshold**: Simple majority of members present
+- **Recording**: Documented in meeting minutes with vote counts
+
+### 2. Asynchronous Votes via GitHub Pull Request
+- **When Used**: 
+  - When quorum cannot be achieved in meetings
```

Comment:
... but the voting should be done before next TSC meeting or a TSC member asks for it because he can not attend a meeting